cmd/security-agent/subcommands/start/command.go

```diff
@@ -314,8 +293,6 @@ func RunAgent(ctx context.Context, log log.Component, config config.Component, t
 		}
 	}()
 
-	demultiplexer.AddAgentStartupTelemetry(fmt.Sprintf("%s - Datadog Security Agent", version.AgentVersion))
```

Comment:
We no longer need this, right?

Reply:
I searched on other agents (like trace agent, process agent) if they were doing this, it looks like they are not. Only the cluster agent does. And everything works perfectly fine without this